### PR TITLE
Feat/bootstrap out

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
@iOiurson Just for you to know, I'm removing Bootstrap (and so, jQuery) from the lib 🎉

The PR is not ready for now, I'll let you know when it's done.

### CODE REVIEW

**Comment by GuillaumeAmat** [comment]
@iOiurson I think I'm done for the first pass. It remains some cleanups but I'll do it by pimping the style to fit the last MapContrib designs.

Can you take a look at the PR? Thanks!

File: package.json [hunk 1]
```diff
@@ -98,9 +97,21 @@
...
+    "@types/enzyme": "^3.1.13",
+    "@types/enzyme-adapter-react-16": "^1.0.3",
+    "@types/extract-text-webpack-plugin": "^3.0.3",
+    "@types/leaflet": "^1.2.9",
+    "@types/react-fontawesome": "^1.6.4",
+    "@types/react-leaflet": "^1.1.6",
+    "@types/react-transition-group": "^2.0.13",
+    "@types/react-virtualized-select": "^3.0.7",
+    "@types/webpack-bundle-analyzer": "^2.9.2",
+    "@types/webpack-node-externals": "^1.6.3",
```
Comment (bleucitron) [inline package.json]: These should only be in the dedicated PR.

File: src/components/Form/Select.jsx [hunk 2]
```diff
@@ -24,17 +24,23 @@ const StyledSelect = styled(VirtualizedSelect)`
...
+      padding-left: 1rem;
     }
 
     .Select-value .Select-value-label {
-      line-height: 3.8rem;
+      line-height: 2.3rem;
+    }
+  }
+
+  &&.Select--multi {
```
Comment (bleucitron) [inline src/components/Form/Select.jsx:41]: Isn't `&` enough (instead of 2) ?

File: src/components/Osm/Editor.jsx [hunk 3]
```diff
@@ -131,53 +123,53 @@ class Editor extends React.Component {
     return '';
   }
 
-  addField(tag, value) {
+  addField = (tag, value) => {
```
Comment (bleucitron) [inline src/components/Osm/Editor.jsx:126]: Oh, i wasn't aware we were using class properties...

**Review by bleucitron — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (GuillaumeAmat) [reply package.json]: Yep, you're right I'll remove that

File: src/components/Form/Select.jsx (hunk 2 above)
Comment (GuillaumeAmat) [reply src/components/Form/Select.jsx:41]: The selector weight is not higher enough with only one.

Comment (GuillaumeAmat) [reply src/components/Form/Select.jsx:41]: ... in that specific case

File: src/components/Osm/Editor.jsx (hunk 3 above)
Comment (GuillaumeAmat) [reply src/components/Osm/Editor.jsx:126]: No worries, it was just to be less verbose

**Comment by GuillaumeAmat** [comment]
@iOiurson Are you ok with that PR?

**Comment by GuillaumeAmat** [comment]
F*** Travis is the worst... 30 minutes timeout is not enough to pass the tests! (see https://travis-ci.org/osm-ui/react/builds/421232316)

Can we merge that PR and move to Gitlab right after? Are you near the end of your PRs?

**Comment by bleucitron** [comment]
Yeah i'm good with that PR, sorry i thought i had approved it, but apparently i had not...

My PRs are good on my side, they just don't pass the CI... not sure if it's only because of the timeout thing...

I'm also ok with moving to Gitlab, but we should merge all 3 pending PRs before that. Can check them out ?

**Review by bleucitron — APPROVED** [review]

**Comment by GuillaumeAmat** [comment]
> Yeah i'm good with that PR, sorry i thought i had approved it, but apparently i had not...

Great thanks!

> My PRs are good on my side, they just don't pass the CI... not sure if it's only because of the timeout thing...

> I'm also ok with moving to Gitlab, but we should merge all 3 pending PRs before that. Can check them out ?

Sure, I'll look at it right now and merge it if it's a timeout problem.
